src/pseudopeople/entity_types.py

```diff
@@ -66,7 +66,12 @@ def __call__(
         additional_key: Any,
     ) -> pd.Series:
         column = column.copy()
-        noise_level = configuration[Keys.PROBABILITY] * self.noise_level_scaling_function(
+        probability_key = (
```

Comment:
Should I add protection that we (engineers) don't accidentally keep both `probability` and `cell_probability`? As-is, if we add `cell_probability` via additional parameters and forget to `probability=None`, things will still work but will default to using "probability"

Reply:
I guess it doesn't hurt but seems like tests would break pretty quickly?

Reply:
There are no tests against this. LIke, if we say in the schema_entities.py that a certain noising function is to have both types of probability, then the configurator will happily build it that way.

Reply:
I can raise a runtime error while building the configuration or something like that.

Reply:
Maybe use cell probability if it exists and then use probability if it doesn't? I can't imagine us accidentally adding cell probability when we don't want it.